step-framework-parent/step-framework-collections/src/main/java/step/core/collections/IndexField.java

```diff
@@ -0,0 +1,38 @@
+package step.core.collections;
+
+public class IndexField {
```

Comment:
Does this have to be mutable? It looks a lot like a struct (or data-only) class that could be immutable, and thus be simplified to just have public final fields.